schema/defs-linux.json

```diff
@@ -93,7 +93,9 @@
             "type": "string"
         },
         "FileType": {
-            "type": "integer"
+            "description": "Type of a block or special character device",
+            "type": "string",
+            "pattern": "^[c|b|u|p]$"
```

Comment:
On Wed, Apr 06, 2016 at 07:38:15PM -0700, Vincent Batts wrote:
> > @@ -93,7 +93,9 @@
> >              "type": "string"
> >          },
> >          "FileType": {
> > -            "type": "integer"
> > +            "description": "Type of a block or special character device",
> > +            "type": "string",
> > +            "pattern": "^[c|b|u|p]$"
> 
> http://json-schema.org/latest/json-schema-validation.html#anchor33

Thanks :).  That references ECMA 262, but [1] is too dense for me.
[2] has:

  [xyz] [a-c] A character set. Matches any one of the enclosed
              characters. You can specify a range of characters by
              using a hyphen, but if the hyphen appears as the first
              or last character enclosed in the square brackets it is
              taken as a literal hyphen to be included in the
              character set as a normal character.

              For example, [abcd] is the same as [a-d]. They match the
              "b" in "brisket" and the "c" in "chop".

And in my browser console:

  >> /^[cbup]$/.test('c');
  ← true
  >> /^[cbup]$/.test('|');
  ← true
  >> /^[c|b|u|p]$/.test('|');
  ← true

So I'm even more sure that we don't want the pipes inside the
character set.

[1]: http://www.ecma-international.org/ecma-262/5.1/#sec-15.10.2.13
[2]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp#character-classes